extensions/udapdr/scripts/colbert/data/ranking.py

```diff
@@ -0,0 +1,94 @@
+import os
```

Comment:
This is very similar to 

./primeqa/extensions/ir_benchmarking/colbert/data/ranking.py
./primeqa/primeqa/ir/dense/colbert_top/colbert/data/ranking.py

Specifically: 
 

> $ diff ./primeqa/extensions/udapdr/scripts/colbert/data/ranking.py  ./primeqa/extensions/ir_benchmarking/colbert/data/ranking.py
37d36
<         # TODO: Handle list of lists???
64c63
<         assert 'tsv' in new_path.strip('/').split('/')[-1].split('.'), "TODO: Support .json[l] too."
---
> \>         assert 'tsv' in new_path.strip('/').split('/')[-1].split('.')

and

> diff ./primeqa/extensions/udapdr/scripts/colbert/data/ranking.py ./primeqa/primeqa/ir/dense/colbert_top/colbert/data/ranking.py
4c4
< from colbert.infra.provenance import Provenance
---
> from primeqa.ir.dense.colbert_top.colbert.infra.provenance import Provenance
6,8c6,8
< from colbert.infra.run import Run
< from colbert.utils.utils import print_message, groupby_first_item
< from utility.utils.save_metadata import get_metadata_only
---
> \>from primeqa.ir.dense.colbert_top.colbert.infra.run import Run
> \>from primeqa.ir.dense.colbert_top.colbert.utils.utils import print_message, groupby_first_item, create_directory
> \>from primeqa.ir.dense.colbert_top.utility.utils.save_metadata import get_metadata_only
64a65
>  \>       create_directory(os.path.dirname(new_path))